.npmignore

```diff
@@ -1,3 +1,20 @@
 # standard ignore
-.gitignore
+*config*
+*conf*
+
 .github/
+.gitignore
+.eslintrc
+.stylelintrc
+
+coverage/
+docs/
+scripts/
+test/
+demo/**/*.css
+demo/**/*.scss
+demo/**/*.html
+
+# ignore src styles
+src/**/*.scss
+src/**/*.css
```

Comment:
Revert these changes. This is not the file that needs to be updated. Please update the following

- https://github.com/AlaskaAirlines/WC-Generator/blob/master/template/.npmignore.temp

Also, please add the ignoring this directory 

- https://github.com/AlaskaAirlines/WC-Generator/tree/master/template/apiExamples

This is a generator repo. Consider the `./template` directory as if it was a `:root` directory.

Reply:
Please update this commit to `fix`. This is a correction to a bug in the config that caused additional resources to be included in the package that was unnecessary.

Reply:
will do, thanks @blackfalcon.